bundles/org.eclipse.passage.lic.mail/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Version: 0.5.0.qualifier
+Bundle-Name: %Bundle-Name
+Bundle-Vendor: %Bundle-Vendor
+Bundle-Copyright: %Bundle-Copyright
+Bundle-SymbolicName: org.eclipse.passage.lic.mail
```

Comment:
Let's move mail interfaces inside lic.net and use this one to isolate javax.mail dependency